Tests/PureeTests/LoggerTests.swift

```diff
@@ -50,6 +48,7 @@ class LoggerTests: XCTestCase {
             ])
         let logger = try! Logger(configuration: configuration)
         logger.postLog(["page_name": "Top", "user_id": 100], tag: "pv")
+        logger.suspend()
```

Comment:
[IMO] How about `suspend` on `tearDown`?
It helps to prevent from forgetting calling `suspend`.

But we have to add a instance property instead... `var logger: Logger!`

Reply:
We have to call `suspend` before `XCTAssertEqual` ...